src/views/Export/ExportSelection/FilterForm/index.tsx

```diff
@@ -0,0 +1,318 @@
+    {
+        key: 'assessment_exists',
+        value: _ts('leads', 'assessmentExistsOptionLabel'),
+    },
+    {
+        key: 'assessment_does_not_exist',
+        value: _ts('leads', 'assessmentDoesNotExistsOptionLabel'),
+    },
+];
+
+interface FilterValues {
+}
+
+interface OwnProps {
+    className?: string;
+    projectId: number;
+    filterOnlyUnprotected?: boolean;
+    filterValues: FilterValues;
+}
+
+const optionLabelSelector = (d: KeyValueElement) => d.value;
+const optionKeySelector = (d: KeyValueElement) => d.key;
+const emmRiskFactorsKeySelector = (d: KeyValueElement) => (isTruthyString(d.key) ? d.key : 'None');
+const emmRiskFactorsLabelSelector = (d: EmmEntity) => (isTruthyString(d.label)
+    ? `${d.label} (${d.totalCount})`
+    : `None (${d.totalCount})`
+);
+
+const emmEntitiesKeySelector = (d: EmmEntity) => d.key;
+const emmEntitiesLabelSelector = (d: EmmEntity) => `${d.label} (${d.totalCount})`;
+const emmTriggerKeySelector = (d: EmmEntity) => d.key;
+const emmTriggerLabelSelector = (d: EmmEntity) => `${d.label} (${d.totalCount})`;
+
+function FilterForm(props: OwnProps) {
+    const {
+        className,
+        projectId,
+        filterValues,
+        onChange,
+        filterOnlyUnprotected,
+    } = props;
+
+    const schema = useMemo(() => ({
+        fields: {
+            search: [],
+            assignee: [],
+            created_at: [],
+            published_on: [],
+            confidentiality: [],
+            status: [],
+            priority: [],
+            authoring_organization_types: [],
+            exists: [],
+            emm_risk_factors: [],
+            emm_keywords: [],
+            emm_entities: [],
+        },
+    }), []);
+
+    const [faramValues, setFaramValues] = useState<FilterValues>(filterValues);
+    const [pristine, setPristine] = useState(true);
+
+    const [
+        pending,
+        leadOptions,
+        ,
+    ] = useRequest<LeadOptions>({
+        url: 'server://lead-options/',
+        query: {
+            projects: [projectId],
+        },
+        autoTrigger: true,
+        method: 'GET',
+        schemaName: 'projectLeadFilterOptions',
+        onSuccess: (response) => {
+            console.warn('here', response);
+        },
+    });
+
+    const {
+        confidentiality,
+        status,
+        assignee,
+        priority,
+        organizationTypes,
+        emmEntities = emptyList,
+        emmKeywords = emptyList,
+        emmRiskFactors = emptyList,
+        hasEmmLeads,
+    } = leadOptions || {};
+
+    const isFilterEmpty = useMemo(() => {
+        let isFilterEmptyValue;
+
+        if (filterOnlyUnprotected) {
+            let newFilter = {
+                ...faramValues,
+                confidentiality: undefined,
+            };
+
+            isFilterEmptyValue = doesObjectHaveNoData(newFilter, ['']);
+
+            newFilter = {
+                ...faramValues,
+                confidentiality: ['unprotected'],
+            };
```

Comment:
what is  use of this?

Reply:
For the viewer role, only unprotected data is available, that is why if filterOnlyUnproteted is true, then we set it my default.

Reply:
here the created newFilter is unused.